packages/app/src/cli/services/app-logs/logs-command/ui/components/Logs.test.tsx

```diff
@@ -164,117 +56,471 @@ STORE_NAME_BY_ID.set('1', 'my-store')
+
+      const renderInstance = render(
+        <Logs
+          pollOptions={{jwtToken: MOCKED_JWT_TOKEN, filters: EMPTY_FILTERS, cursor: MOCKED_CURSOR}}
+          resubscribeCallback={vi.fn().mockResolvedValueOnce(MOCKED_JWT_TOKEN)}
+          storeNameById={new Map()}
+        />,
+      )
+
+      const lastFrame = renderInstance.lastFrame()
+
+      expect(unstyled(lastFrame!)).toMatchInlineSnapshot(`
+      "2024-06-18 16:02:04.868 my-store my-function Success export \\"run\\" executed in 0.5124M instructions
+          test logs
+
+          Input Query Variables:
+
+           Namespace: inputQueryVariablesMetafieldNamespace
+           Key: inputQueryVariablesMetafieldKey
+
+           No metafield data found!
+
+          Input (10 bytes):
+
+           {
+             \\"test\\": \\"input\\"
+           }
+
+          Output (10 bytes):
+
+           {
+             \\"test\\": \\"output\\"
+           }"
+      `)
+
+      renderInstance.unmount()
+    })
+
+    test('redners FunctionRunLog without iqv when key and namespace are null', async () => {
+      const appLogOutput = {
+        appLog: new FunctionRunLog({
+          export: 'run',
+          input: INPUT,
+          inputBytes: INPUT_BYTES,
+          output: OUTPUT,
+          outputBytes: OUTPUT_BYTES,
+          logs: LOGS,
+          functionId: FUNCTION_ID,
+          fuelConsumed: FUEL_CONSUMED,
+          errorMessage: 'errorMessage',
+          errorType: 'errorType',
+          inputQueryVariablesMetafieldValue: null,
+          inputQueryVariablesMetafieldNamespace: null,
+          inputQueryVariablesMetafieldKey: null,
+        }),
+        prefix: {
+          functionId: FUNCTION_ID,
+          logTimestamp: TIME,
+          description: `export "run" executed in 0.5124M instructions`,
+          storeName: 'my-store',
+          status: STATUS === 'success' ? 'Success' : 'Failure',
+          source: SOURCE,
+        },
+      }
+
+      const mockedUsePollAppLogs = vi.fn().mockReturnValue({appLogOutputs: [appLogOutput], errors: []})
+      vi.mocked(usePollAppLogs).mockImplementation(mockedUsePollAppLogs)
+
+      const renderInstance = render(
+        <Logs
+          pollOptions={{jwtToken: MOCKED_JWT_TOKEN, filters: EMPTY_FILTERS, cursor: MOCKED_CURSOR}}
+          resubscribeCallback={vi.fn().mockResolvedValueOnce(MOCKED_JWT_TOKEN)}
+          storeNameById={new Map()}
+        />,
+      )
+
+      const lastFrame = renderInstance.lastFrame()
+
+      expect(unstyled(lastFrame!)).toMatchInlineSnapshot(`
+      "2024-06-18 16:02:04.868 my-store my-function Success export \\"run\\" executed in 0.5124M instructions
+          test logs
+
+          Input (10 bytes):
+
+           {
+             \\"test\\": \\"input\\"
+           }
+
+          Output (10 bytes):
+
+           {
+             \\"test\\": \\"output\\"
+           }"
+      `)
+
+      renderInstance.unmount()
+    })
+  })
+
+  describe('Webhooks', () => {
```

Comment:
I'm surprised that this is called webhooks- shouldn't it be 'Network Access'?

Reply:
Ah right, will update this :)